tokens/_aliases.yml

```diff
@@ -183,17 +183,17 @@ aliases:
 
   # Content
   content-xs: '304px'
-  content-sm: '464px'
-  content-md: '736px'
-  content-lg: '992px'
-  content-xl: '1248px'
+  content-sm: '432px'
+  content-md: '688px'
+  content-lg: '928px'
+  content-xl: '1152px'
   content-max: '1420px'
```

Comment:
So, sounds like this is changing to 1440px?

Reply:
Updated 👍